bigbluebutton-html5/imports/startup/client/auth.js

```diff
@@ -88,15 +82,6 @@ export function authenticatedRouteHandler(nextState, replace, callback) {
     .then(callback)
     .catch((reason) => {
       log('error', reason);
```

Comment:
Like stated on the PR description, we don't (and shouldn't) control the leaving in the client-side. The server will trigger a `UserLeaveReqMsg` when the connection close.